app/helpers/truncate-text.js

```diff
@@ -0,0 +1,19 @@
+import {helper} from '@ember/component/helper';
+
+function truncateText(params, hash) {
+  const [ value ] = params;
+  const { limit } = hash;
+  let text = '';
+
+  if (value != null && value.length > 0) {
```

Comment:
If `value` is an number, this will crash. I'd ensure `value` is a string here (logically it should, but it's good to be safe).

Reply:
It should break if it is not a text or null.
It is created specifically for a text.

Reply:
@bekicot Is the value passed by Ember always a string?

If not then it's better to do what andrew said, just add typeof check, it's better to return an empty string rather than an error that may cause the app to halt.

Reply:
It is not ember who is using this helper, it is us. If ember change our string into number without us teling ember to do so, then, that might be an error or a bug. Worth to know.

We shall never pass a number when it is expected to be a string. Hence, letting us know that it is wrong is helpful in my opinion.

Anyway, okay, I will change it.

Reply:
Okay – I'm not a pro at Ember and was not sure if this is something we are using or Ember is using, but if we use it and we're sure a number won't be passed, then I guess it's fine to keep as is. However, with that being said, why are we checking for `null` if we're always going to pass in a string? I feel like it should be all or nothing here, so it doesn't make sense to check for `null` and not check for a number. I'd like to hear counter arguments, though.

Reply:
Well, I will change it in the next push.

The I use the concept of null and exist as the one in swift `optionals` and Typescript `String?`.